api_tests/wb/views/test_wb_hooks.py

```diff
@@ -0,0 +1,608 @@
+from __future__ import unicode_literals
+
+import pytest
+
+from addons.osfstorage import settings as osfstorage_settings
+from addons.osfstorage.models import OsfStorageFileNode, OsfStorageFolder
+from framework.auth import signing
+
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    UserFactory,
+)
+from api_tests.utils import create_test_file
+from osf.models import QuickFilesNode
+
+
+@pytest.mark.django_db
+class HookTestCase():
```

Comment:
Now that we're using pytest fixtures, we can usually avoid inheritance in our tests (which can make tests more complicated than they need to be).

So instead of having `HookTestCase`, you can turn the method fixtures below into module-level function fixtures. `sign_payload` can also be a function instead of a method. Then `HookTestCase` can go away!